src/content/en/2021/security.md

```diff
@@ -1099,54 +1104,58 @@ We now turn to cryptominer market share in the following figure.
 {{ figure_markup(
   image="security-cryptominer-market-share.png",
   caption="Cryptominer market share (mobile).",
-  description="Pie chart showing Coinimp has 84.9% of market share, CoinHive has 9.0%, JSEcoin has 3.1%, Minero.cc has 1.5% and others have 1.5%.",
+  description="Pie chart showing CoinImp has 84.9% of market share, CoinHive has 9.0%, JSEcoin has 3.1%, Minero.cc has 1.5% and others have approximately 1.5%.",
   chart_url="https://docs.google.com/spreadsheets/d/e/2PACX-1vR38ZfiZXxdGuzG4ywCEPKIU1Wl6E2bZwcQPyALavGq5q948gmWb8sT-Xo5T6K5z8smKPg6EKxV0JUI/pubchart?oid=1688060716&format=interactive",
   sheets_gid="1998142066",
   sql_file="cryptominer_share.sql"
 ) }}
 
-We see that [CoinHive](https://en.wikipedia.org/wiki/Coinhive) community has switched to CoinImp. Therefore, CoinImp has clearly become the market leader (90% share). Even though Coinhive has been discontinued, we still see some websites that use CoinHive scripts (Desktop: 5.7%, mobile: 9%).
+We see that [CoinHive](https://en.wikipedia.org/wiki/Coinhive) community has switched to CoinImp. Therefore, CoinImp has clearly become the market leader (84.9% share). Even though Coinhive has been discontinued, we still see some websites that use CoinHive scripts (Desktop: 5.7%, mobile: 9.0%).
```

Comment:
1) the Wikipedia page redirects to Monero, so not too sure how useful this link is (in the least, link to https://en.wikipedia.org/wiki/Monero#Mining_malware, but maybe a link won't be necessary, because:)
2) it might be interesting to provide the context that the Coinhive domain is now [owned by Troy Hunt](https://www.troyhunt.com/i-now-own-the-coinhive-domain-heres-how-im-fighting-cryptojacking-and-doing-good-things-with-content-security-policies/), and that he has set it up to display a banner (even becoming [quite aggressive](https://twitter.com/troyhunt/status/1455658641344761860))
3) capitalization nit (also for the chart):
```suggestion
We see that [Coinhive](https://en.wikipedia.org/wiki/Coinhive) community has switched to CoinImp. Therefore, CoinImp has clearly become the market leader (84.9% share). Even though Coinhive has been discontinued, we still see some websites that use Coinhive scripts (Desktop: 5.7%, mobile: 9.0%).
```

Reply:
Corrected the chart.

Though these names are wrong in Wappalyzer too so probably should correct there?

@SaptakS could you look at the wording? Link to the Troy Hunt post would be good IMHO.

Reply:
I have asked @nrllh to take a look at this since he wrote this section.

Reply:
As discussed on Slack, here's the suggested wording:

```suggestion
We see that [Coinhive](https://en.wikipedia.org/wiki/Monero#Mining_malware) has been surpassed by CoinImp as the dominate cryptomining service. One of the main reasons for this <a hreflang="en" href="https://www.zdnet.com/article/coinhive-cryptojacking-service-to-shut-down-in-march-2019/">was that Coinhive was shoutdown in March 2019</a>. Interestingly, the domain is now <a hreflang="en" href="https://www.troyhunt.com/i-now-own-the-coinhive-domain-heres-how-im-fighting-cryptojacking-and-doing-good-things-with-content-security-policies/">owned by Troy Hunt</a> who is now displaying aggressive banners in the website in effort to make those sites still hosting the Coinhive script (Desktop: 5.7%, mobile: 9.0%) aware that they are—often without their knowledge. This reflects both the prevalence of Coinhive scripts even over two years after ceasing to operate, and the risks of hosting third-party resources that can be taken over should that third party cease to operate. With Coinhive's demise, CoinImp has clearly become the market leader (84.9% share).
```

Reply:
Updated the section with the suggestions. Thanks @tunetheweb and @VictorLeP !